x/profiles/client/cli/tx.go

```diff
@@ -30,6 +31,8 @@ func GetTxCmd(_ string, cdc *codec.Codec) *cobra.Command {
 	profileTxCmd.AddCommand(flags.PostCommands(
 		GetCmdSaveProfile(cdc),
 		GetCmdDeleteProfile(cdc),
+		GetCmdCreateMonoDirectionalRelationship(cdc),
+		GetCmdDeleteUserRelationship(cdc),
```

Comment:
What about adding a new command to query all the relationship for all the profiles?